src/lib/viewers/controls/color-picker/ColorPickerPalette.scss

```diff
@@ -0,0 +1,46 @@
+.bp-ColorPickerPalette {
+    position: absolute;
+    left: -55px;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 35px;
+    margin-bottom: 125px;
```

Comment:
We should try to avoid this by using a `top` value that positions the palette above its control bar.

Reply:
Sounds good, I can use this instead:

```
top: -50px;
```